Samples/kotlin/src/main/java/com/phunware/kotlin/sample/routing/util/ManeuverDisplayHelper.kt

```diff
@@ -102,18 +104,25 @@ class ManeuverDisplayHelper {
         return String.format(Locale.US, context.getString(R.string.section_distance), prep, getStringDistanceInFeet(maneuver.distance))
     }
 
+    private fun getDistanceInFeet(distance: Double): Double {
+        if (distance < 0) {
+            return -1.0
+        }
+        val distInFeet = distance * NUM_FEET_PER_METER
+        return Math.ceil(distInFeet)
+    }
+
     /**
      * Converts the distance from meters to feet
      * @param distance Double distance in meters
      * @return String object containing the converted number of feet (rounded up)
      */
     private fun getStringDistanceInFeet(distance: Double): String {
         if (distance < 0) {
-            return ""
+            return "";
+        } else {
+            return getDistanceInFeet(distance).toInt().toString();
```

Comment:
Another option here would be:
DecimalFormat().apply {
    isDecimalSeparatorAlwaysShown = false
}.format(getDistanceInFeet(distance)

A little more verbose but very descriptive to what your are trying to achieve.

Reply:
Good suggestion. We can add in the next update, since this build has already been given to QA.